doc/tutorial_1.rst

```diff
@@ -596,10 +600,10 @@ Sub-tasks (items of task group) by default are not reported on ``list`` command.
    imports:requests.utils
 
 
-Note the task name is composed by the (base name) group task name
```

Comment:
Here I have not introduced it, but a `doit` task might have an attribute called `basename`.
So I would like to keep consistent with `doit` terminology.
I suggest we use your alteration but still mentions "basename", something like:
 
Note the task name is composed by the task's group name (also known as ``basename``).